resources/metadata/sonar.yaml

```diff
@@ -41,6 +41,10 @@ spec:
           - PARAMETERS
         secret: true
         resourceRef:
+          - type: vaultSecret
```

Comment:
Jenkins Secrets are maintained as `spec.input.secret` of type `jenkins`, shouldn't we implement vault the same ways with type `vault`?

Reply:
@OliverNocon